jsonexamples/CMakeLists.txt

```diff
@@ -1,5 +1,13 @@
-set(SIMDJSON_BENCHMARK_DATA_DIR ${CMAKE_CURRENT_SOURCE_DIR} PARENT_SCOPE)
-set(EXAMPLE_JSON ${CMAKE_CURRENT_SOURCE_DIR}/twitter.json PARENT_SCOPE)
-set(EXAMPLE_NDJSON ${CMAKE_CURRENT_SOURCE_DIR}/amazon_cellphones.ndjson PARENT_SCOPE)
+set(SIMDJSON_BENCHMARK_DATA_DIR ${CMAKE_CURRENT_BINARY_DIR} PARENT_SCOPE)
+set(EXAMPLE_JSON ${CMAKE_CURRENT_BINARY_DIR}/twitter.json PARENT_SCOPE)
+set(EXAMPLE_NDJSON ${CMAKE_CURRENT_BINARY_DIR}/amazon_cellphones.ndjson PARENT_SCOPE)
 add_library(jsonexamples-data INTERFACE)
-target_compile_definitions(jsonexamples-data INTERFACE SIMDJSON_BENCHMARK_DATA_DIR="${CMAKE_CURRENT_SOURCE_DIR}/")
+target_compile_definitions(jsonexamples-data INTERFACE SIMDJSON_BENCHMARK_DATA_DIR="${CMAKE_CURRENT_BINARY_DIR}/")
+
+# Copy static files to the build dir so they live alongside the generated ones
```

Comment:
>  Copy static files to the build dir so they live alongside the generated ones

I understand, but what is the rationale? These files do not get released, right?

Reply:
Rationale is so that we can have just one data directory. Since we generate files into the build/ directory rather than top-level, all json we want to use needs to go to that directory if we want to use it. We could definitely just have two variables, too.

Reply:
Ok.